### PR TITLE
Pair-Programming Local storage

### PR DESCRIPTION
**Pull request by principlebrothers**
Add functions in localStorage.js to locally store user entries in the form'

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi @principlebrothers , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is supposed to be worked on in pairs,  your team is supposed to contributed to this project as well .

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: JavaScript/locaStorage.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+}
+
+function getData(){
+  const receivedData = localStorage.getItem('form');
+  if(receivedData) {
+    const formObject = JSON.parse(receivedData);
+    emailInput.value = formObject.email;
+    message.value = formObject.msg;
+    inputName.value = formObject.name;
+  }
```
Comment (roshan-bajgain) [inline JavaScript/locaStorage.js]: - [ ] After filling out the form, and not submitting it, the entered values should be saved in the browser. Kindly implement it.

Comment (roshan-bajgain) [inline JavaScript/locaStorage.js]: - [ ] The contact form information is persistent to page reloads Kindly implement it

Comment (roshan-bajgain) [inline JavaScript/locaStorage.js]: - [ ] When I fill the form the value is not stored in local storage kindly implement it.

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @principlebrothers, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by koachcode — APPROVED** [review]
Hi @principlebrothers , 

# STATUS: APPROVED 🟢

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- You set up linter files and no errors. 💯 

well-1

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**